designer/server/src/test/scala/pl/touk/nussknacker/ui/statistics/UsageStatisticsReportsSettingsDeterminerTest.scala

```diff
@@ -170,34 +214,80 @@ class UsageStatisticsReportsSettingsDeterminerTest
       isFragment = false,
       ProcessingMode.UnboundedStream,
       DeploymentManagerType("flinkStreaming"),
-      Some(SimpleStateStatus.NotDeployed)
+      Some(SimpleStateStatus.NotDeployed),
+      nodesCount = 2,
+      scenarioCategory = "Category1",
+      scenarioVersion = VersionId(2),
+      createdBy = "user",
+      fragmentsUsedCount = 0,
+      lastDeployedAction = None
     )
     val runningScenario = ScenarioStatisticsInputData(
       isFragment = false,
       ProcessingMode.UnboundedStream,
       DeploymentManagerType("flinkStreaming"),
-      Some(SimpleStateStatus.Running)
+      Some(SimpleStateStatus.Running),
+      nodesCount = 2,
+      scenarioCategory = "Category1",
+      scenarioVersion = VersionId(2),
+      createdBy = "user",
+      fragmentsUsedCount = 0,
+      lastDeployedAction = None
     )
     val fragment = ScenarioStatisticsInputData(
       isFragment = true,
       ProcessingMode.UnboundedStream,
       DeploymentManagerType("flinkStreaming"),
-      None
+      None,
+      nodesCount = 2,
+      scenarioCategory = "Category1",
+      scenarioVersion = VersionId(2),
+      createdBy = "user",
+      fragmentsUsedCount = 0,
+      lastDeployedAction = None
     )
     val k8sRRScenario = ScenarioStatisticsInputData(
       isFragment = false,
       ProcessingMode.RequestResponse,
       DeploymentManagerType("lite-k8s"),
-      Some(SimpleStateStatus.Running)
+      Some(SimpleStateStatus.Running),
+      nodesCount = 2,
+      scenarioCategory = "Category1",
+      scenarioVersion = VersionId(2),
+      createdBy = "user",
+      fragmentsUsedCount = 0,
+      lastDeployedAction = None
     )
 
     val params = new UsageStatisticsReportsSettingsDeterminer(
       UsageStatisticsReportsConfig(enabled = true, Some(sampleFingerprint), None),
       fingerprintSupplier,
-      () => Future.successful(List(nonRunningScenario, runningScenario, fragment, k8sRRScenario))
+      () => Future.successful(List(nonRunningScenario, runningScenario, fragment, k8sRRScenario)),
+      () => Future.successful(processActivityList),
+      () => Future.successful(componentList)
     ).determineQueryParams().futureValue
 
     val expectedStats = Map(
+      "a_n"     -> "1",
```

Comment:
For better readability I would replace the string keys with the ADT, fe:
`AuthorsCount.name -> "1"`